### DISCUSSION TITLE
Failed to install npm packages when running

### DISCUSSION THREAD

**Discussion by leojamesstrong** [ArchiveBox/ArchiveBox#1218] [Q&A]
Hi! I've been trying to set up ArchiveBox on Windows 11 using `pip`.

When I run `archivebox init --setup` I get this error:

`Installing SINGLEFILE_BINARY, READABILITY_BINARY, MERCURY_BINARY automatically using npm...
←[01;31m[X] Failed to install npm packages: [WinError 2] The system cannot find the file specified←[00;00m
    ←[01;35mHint:←[00;00m Try deleting C:\Users\leofs\Dropbox\ArchiveBox/node_modules and running it again`

I'm not sure what the issue is, I think I installed all the dependencies? Please let me know if there's a solution to this I'm missing!

**Comment by pirate** [comment]
Sorry I can't provide windows support for bare metal, you should run it in Docker or in the WSL2.
